### PR TITLE
Add feedback to pyth pathway

### PR DESCRIPTION
**Pull request by dgamboa**

### CODE REVIEW

File: markdown/pyth/PYTH_LIQUIDATE.md [hunk 1]
```diff
@@ -40,3 +40,5 @@ It is also handy to have a peek at the dev tools console in your browser to see
 # 🏁 Conclusion
 
 Congratulations, you now have a very basic yet functional liquidation bot which depends on Pyth price data to make token swaps on a DEX! There are many ways in which you might seek to improve on this functionality - whether it's just an improved front-end design or implementing a more stringent algorithm for trading.
+
+<!-- Why does the total worth toggle on and off? Also the % change didn't change for me, am I missing something? -->
```
Comment (pyr0gan) [inline markdown/pyth/PYTH_LIQUIDATE.md:44]: @Necmttn should be able to fix the % change

File: markdown/pyth/PYTH_VISUALIZE_DATA.md [hunk 2]
```diff
@@ -232,6 +232,8 @@ export const Chart: React.FC<{data: any}> = ({data}) => {
 
 The `CustomizedHistoricalHourAxisTick` is used to display the vertically oriented timestamps at the bottom of the chart. These are what we'll see for the short period of time we're running the chart here. `CustomizedHistoricalDayAxisTick` is used when displaying the larger time ranges. The chart has a dropdown menu to select the time range for displaying data - however we are not storing the chart data anywhere in this app and so the DAY and WEEK settings are mainly for illustrative purposes.
 
+<!-- why have the drodown at all if it's just for illustrative purposes? Why not have just a label that says live? If it's part of the standard component, then it's fine to leave it as is. But if it's easy to change, we should probably change it for simplicity. -->
```
Comment (pyr0gan) [inline markdown/pyth/PYTH_VISUALIZE_DATA.md:235]: @Necmttn What do you think? Should we maybe cut out the larger display windows? 😄

Comment (Necmttn) [reply markdown/pyth/PYTH_VISUALIZE_DATA.md:235]: Yeah, we can remove it, I was applying directly Pyth teams chart code there, I agree with @dgamboa.

File: markdown/pyth/PYTH_LIQUIDATE.md (hunk 1 above)
Comment (Necmttn) [reply markdown/pyth/PYTH_LIQUIDATE.md:44]: fixed 🎉

**Pull request closed without merging**